bingo.js

```diff
@@ -453,7 +488,8 @@ function toggleStreamerMode()
 	$(".dropdown").hide();
 	hideOptionsMenu();
 	updateStreamerMode();
-	pushNewUrl();
+	pushNewUrl(generateNewUrlParamWithProgress());
```

Comment:
I don't know JavaScript conventions, but it might be better if missing parameter was explicitly passed as `undefined` or `null` here.

Reply:
Only if JavaScript allowed default parameters values e.i.:
function generateNewUrlParamWithProgress(squareProgress = null) { ... }

Reply:
There is 'default parameter value' to be set in function declaration as `function generateNewUrlParamWithProgress(param = defaultValue) { ... }`.

But I tend to stick with 'default parameter value' to be checked as `if (parameter === undefined) { parameter = defaultValue; }` inside the function, since I personally still use older version of safari, that doesn't support ES6 (https://caniuse.com/?search=es6), meaning the whole page wouldn't work :(

I'll pass null for better readability.

Reply:
Actually, I see that there are some usage of arrow functions, meaning my point isn't really valid here.
Will update to 'default parameter value' to be set in function declaration.